contracts/WildcardSteward_v1.sol

```diff
@@ -292,6 +267,7 @@ contract WildcardSteward_v1 is Initializable {
         } else if(state[tokenId] == StewardState.Foreclosed) {
             state[tokenId] = StewardState.Owned;
             timeLastCollected[tokenId] = now;
+            timeLastCollectedPatron[msg.sender] = now;
```

Comment:
This line is super critical, basically without it you can find a way to freeze the tokens if the foreclose with a high enough price and enough time passes.